compiler/rustc_hir_typeck/src/method/prelude_edition_lints.rs

```diff
@@ -32,22 +32,30 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
             segment.ident, self_ty, call_expr, self_expr
```

Comment:
Method is called `lint_dot_call_from_2018`, I don't think `2018` is still accurate?